README.md

```diff
@@ -46,21 +46,21 @@ We don't have any particular recomendation about hosting providers for tellaweb:
 
 
 ## Updating and Maintainance
-As a recomendation you should check regularly for new releases of tellaweb in our [releases page](https://github.com/Horizontal-org/tellaweb/releases).
+We recommend monitoring our [releases page](https://github.com/Horizontal-org/tellaweb/releases) and updating Tella Web when possible to ensure you get the latest security improvements and new features. 
 
-for updating tellaweb we recommend:
+To update Tella Web, we recommend:
 
-1. Doing a backup of the database
```

Comment:
@juandans01 can we give more info here, like a recommendation on how to do it or what tool to use? or is it obvious and unnecessary?

Reply:
We have an Infuse deliveable for doing exaclty that, once we do that we can update it here. ok? let's not do this chage now, as we will do it properly in the next few days

Reply:
Yes sounds good!